app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
-
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car):
+        if self.clean_power > car.clean_mark:
+            price = (car.comfort_class *
+                     (self.clean_power - car.clean_mark) *
+                     self.average_rating / self.distance_from_city_center)
+            return round(price, 1)
+        return 0.0
+
+    def wash_single_car(self, car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
```

Comment:
The `wash_single_car` method correctly updates the `clean_mark` of the car, but it might be beneficial to return a value indicating whether the car was washed or not for better clarity and potential future use.